### PR TITLE
mentorship meeting 06-22-2018

### LINKED ISSUE #67: Node.js Mentorship team Meeting 2018-06-22
## Time
**UTC Fri 22-Jun-2018 15:00 (03:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Fri 22-Jun-2018 09:00 (08:00 AM) |
| US / Mountain | Fri 22-Jun-2018 09:00 (09:00 AM) |
| US / Central  | Fri 22-Jun-2018 10:00 (10:00 AM) |
| US / Eastern  | Fri 22-Jun-2018 11:00 (11:00 PM) |
| London        | Fri 22-Jun-2018 16:00 (04:00 PM) |
| Amsterdam     | Fri 22-Jun-2018 17:00 (05:00 PM) |
| Moscow        | Fri 22-Jun-2018 18:00 (06:00 PM) |
| Chennai       | Fri 22-Jun-2018 20:30 (08:30 PM) |
| Hangzhou      | Fri 22-Jun-2018 22:00 (10:00 PM) |
| Tokyo         | Sat 23-Jun-2018 00:00 (12:00 AM) |
| Sydney        | Sat 23-Jun-2018 01:00 (01:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Community%20Committee+Meeting+2018-06-22&iso=20180622T15
* or http://www.wolframalpha.com/input/?i=03PM+UTC%2C+Jun+22%2C+2018+in+local+time

# Agenda
The agenda comes from issues labelled with `mentorship-agenda` across all of the repositories in the nodejs org. Please label any additional issues that should be on the agenda before the meeting starts.

* Follow up discussion for 2018-06-15 meeting #66 
* Identify Mentees Who Could Help Out As "Teaching Assistant"? #49 

# Invited
CommComm Members: @nodejs/community-committee
Mentorship Team: @nodejs/mentorship 
Applied Mentors
- @nickParis11 (for TA discussion)

# Join the meeting
Working Zoom link: https://zoom.us/j/416501153

### PR DESCRIPTION
**Pull request by Bamieh**
closes #67

### CODE REVIEW

File: meeting/06-22-2018.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+## Present
+
+- Dan Shaw ([@dshaw](https://github.com/dshaw) - CommComm member, User Feedback champion)
+- Ahmad Bamieh ([@Bamieh](https://github.com/Bamieh))
+- A.J. Roberts ([@detrohutt](https://github.com/detrohutt)
+- Agiri Abraham ([@codeekage](https://github.com/codeekage))
+- Bayo Opesanaya
+- kostas k
+
```
Comment (detrohutt) [inline meeting/06-22-2018.md]: Justin was present, and possibly one more person but I can't remember.

File: meeting/06-22-2018.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+* **Previous Minutes:** 
+
+## Present
+
+- Dan Shaw ([@dshaw](https://github.com/dshaw) - CommComm member, User Feedback champion)
+- Ahmad Bamieh ([@Bamieh](https://github.com/Bamieh))
+- A.J. Roberts ([@detrohutt](https://github.com/detrohutt)
+- Agiri Abraham ([@codeekage](https://github.com/codeekage))
+- Bayo Opesanaya
+- kostas k
```
Comment (kapekost) [inline meeting/06-22-2018.md]: Kostas Kapetanakis ( @kapekost )

File: meeting/06-22-2018.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+### Agenda
+
+Extracted from mentorship-agenda labelled issues and pull requests from the mentorship repo prior to the meeting.
+
+- Follow up discussion for 2018-06-15 meeting #66
+- Identify Mentees Who Could Help Out As "Teaching Assistant"? #49
+
+**TOPIC**
+
```
Comment (detrohutt) [inline meeting/06-22-2018.md:28]: Please add that Justin (@jxm262) agreed to join the Mentorship Team. Everything else LGTM.

**Review by detrohutt — CHANGES_REQUESTED** [review]

File: meeting/06-22-2018.md (hunk 1 above)
Comment (detrohutt) [reply meeting/06-22-2018.md]: Valerio was the other person besides Justin.

File: meeting/06-22-2018.md (hunk 3 above)
Comment (Bamieh) [reply meeting/06-22-2018.md:28]: updated

Comment (jxm262) [reply meeting/06-22-2018.md:28]: 🙌

Comment (detrohutt) [reply meeting/06-22-2018.md:28]: I added a commit. I'm happy with everything now. :)

**Review by detrohutt — APPROVED** [review]

**Pull request merged**